docs/src/pages/components/data-grid/events/SubscribeToEvents.tsx

```diff
@@ -13,11 +17,14 @@ export default function SubscribeToEvents() {
   });
 
   React.useEffect(() => {
-    return apiRef.current.subscribeEvent('columnResize', (params) => {
-      setMessage(
-        `Column ${params.colDef.headerName} resized to ${params.computedWidth}px.`,
-      );
-    });
+    return apiRef.current.subscribeEvent(
+      'columnResize',
+      (params: GridColumnResizeParams) => {
+        setMessage(
+          `Column ${params.colDef.headerName} resized to ${params.width}px.`,
```

Comment:
Good catch. 👍

Reply:
I was the one introducing it though :grimacing: